src/lib.rs

```diff
@@ -36,10 +37,9 @@ pub mod prelude {
 /// This module contains types representing the strength of an address's reference counting, which
 /// influences whether the address will keep the actor alive for as long as it lives.
 pub mod refcount {
-    pub use crate::inbox::tx::TxEither as Either;
-    pub use crate::inbox::tx::TxRefCounter as RefCounter;
```

Comment:
I would like to keep this here, as it allows users to write generic code like
```rust
trait AddressExt { .. }
impl<A: Actor, Rc: RefCounter> AddressExt for Address<A, Rc> { .. }
```

Otherwise, you need to provide this only for `Address<A, TxWeak>`, which compels the use of `.into_either`

Reply:
A wild requirement appears! 😁

I hadn't thought of this but it makes sense.

It would still be possible to abstract over all addresses, we'd just need to implement it multiple times but it is a bit repetitive.

Let me add a test/example for this in another PR so it's documented and I'll think about how to solve it here.

The unfortunate consequence of exporting the trait is we need to seal it, which adds complexity again :/

Reply:
Hm... I wouldn't mind sealing it too much, to be honest, given that it's not a *super* huge extra complexity to handle. It is a little annoying, though.

Reply:
> Let me add a test/example for this in another PR so it's documented and I'll think about how to solve it here.

https://github.com/Restioson/xtra/pull/169I've opened a PR here.